### ISSUE TITLE
Add VBDec

### ISSUE THREAD

**Issue by buzzer-re** [mentebinaria/retoolkit#64]
VBDec is a amazing tool for Visual Basic P-Code disassembling and debugging, it's free and very powerful! 

http://sandsprite.com/vbdec/

**Comment by Bang1338** [comment]
no native?

**Comment by merces** [comment]
Any ideas on how to automate VBDec's installation? I could not a find way do to so... :(

**Comment by Bang1338** [comment]
> Any ideas on how to automate VBDec's installation? I could not a find way do to so... :(

repack it i guess?

**Comment by merces** [comment]
Yeah it should be possible, but I don't have the time to do it at this point. If someone sends a PR, I'll be more than happy to add it. 🙂

**Comment by InstinctEx** [comment]
I found this program on the web [https://www.silentinstall.org/](https://www.silentinstall.org/ )
seems quite effective for programs that don't support /s or other parameters for silent installs

**Comment by merces** [comment]
I didn't have time to work on this. Will try to do something for the next release. I'll keep the issue open. Thanks!

**Comment by ExeinfoASL** [comment]
**Exeinfo Pe  - detect :  Silent Install Builder v.6.2.5/6  used ( NSIS Installer 3.0.x )**
